src/popup.html

```diff
@@ -18,13 +18,13 @@
   <script id="template" type="x-tmpl-mustache">
     <div class='envs'>
       {{#environments}}
-        <a href="{{url}}" class="{{class}}">{{name}}</a>
+        <a href="{{url}}" class="in-popup {{class}}">{{name}}</a>
```

Comment:
Maybe: "js-rerender-popup"

Reply:
Thanks, I've also updated the external classes which were only used for click handlers to use the `js` prefix too.